score/liquid_icx/liquid_icx.py

```diff
@@ -208,7 +224,11 @@ def join(self) -> None:
         """
 
         if self.msg.value < self._min_value_to_get_rewards.get():
-            revert("Joining value cannot be less than the minimum join value")
+            revert("LiquidICX: Joining value cannot be less than the minimum join value")
+
+        if self._cap.get() < self.getStaked():
```

Comment:
If there's only this if check, this could happen:
`self.getStaked` is at e.g. 990.
Somebody joins with 1 Mio ICX.
He can join, only the next person after cannot.
Then there's much more ICX in the SCORE than the cap of 1000 ICX.

Reply:
So what is your proposal?